File: T4.1.1_ModeladoRelacionesClasesPolimorfismo/Ecosystem/Procariota.java
public class Procariota extends Celula{
    private boolean nucleo = false;

    public boolean getNucleo() {
        return nucleo;
    }

    public void setNucleo(boolean nucleo) {
        this.nucleo = nucleo;
    }

    public String toString(){
        return super.toString() + "Existe nucleo? "+nucleo+"\n";
    }    
}

File: T4.1.1_ModeladoRelacionesClasesPolimorfismo/Ecosystem/ecosystemcs/Procariota.cs
public class Procariota : Celula{
    private Boolean nucleo = false;

    public Boolean getNucleo() {
        return nucleo;
    }

    public void setNucleo(Boolean nucleo) {
        this.nucleo = nucleo;
    }

    public new String toString(){
        return base.toString() + "Existe nucleo? "+nucleo+"\n";
    }    
}